Realm/RLMAccessor.mm

```diff
@@ -888,6 +890,83 @@ static auto to_optional(__unsafe_unretained id const value, Fn&& fn) {
     return to_optional(v, [&](__unsafe_unretained NSUUID *v) { return [v rlm_uuidValue]; });
 }
 
+#pragma mark - RLMStatelessAccessorContext::unbox -
```

Comment:
This appears to be adding a second copy of the functions rather than moving them from RLMAccessorContext to RLMStatelessAccessorContext.

Reply:
I have moved the specialisations to RLMStatelessAccessorContext